pkg/ali/components/ArrayListOrdered.vue

```diff
@@ -1,153 +1,138 @@
-<script>
+<script setup>
```

Comment:
Is there a limitation that is preventing us from using typescript in this component?